genpass/password.py

```diff
@@ -98,15 +103,19 @@ def createpass():
     creation_date = date.today()
     email = click.prompt("Enter email id", default="None")
     portal_url = click.prompt("Enter portal url", default="None")
+    request = requests.get(portal_url)
+
+    if request.status_code != 200:
+        print('Web site does not reachable')
```

Comment:
Since this above code is repeated twice, we can move it to a function and call it here.

Reply:
We are going to close this issue #46 as discussed with,
@ganeshhubale @raukadah @Yadnyawalkya 
As, it's not useful in the future!